SampleApps/QuickStart-SwiftUI/QuickStart/ViewModels/NodeViewModel.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  NodeViewModel.swift
+//  QuickStart
+//
+// Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+import FRAuth
+
+@MainActor class NodeViewModel: ObservableObject {
+    @Published var callbackViewModels = [CallbackViewModel]()
+    @Published var showNodeView = false
+    @Published var shouldDismiss: Bool = false
+    
+    var statusViewModel: StatusViewModel?
+    var journeyName: String = "\(FRAuth.shared?.options?.authServiceName ?? "") Journey" 
+    var node: Node?
+    
+    init(node: Node? = nil) {
+        self.node = node
+    }
+    
+    func parseNode() {
+        guard let node = node else {
+            print("Node is missing")
+            return
+        }
+        
+        for callback in node.callbacks {
+            if callback.isKind(of: NameCallback.self), let nameCallBack = callback as? NameCallback {
```

Comment:
Why do you check both `callback.isKind(of: NameCallback.self)` and `let nameCallBack = callback as? NameCallback`

The latter would be enough and simple as it does the casting as well